bot/exts/moderation/incidents.py

```diff
@@ -21,6 +23,12 @@
 # Seconds for `crawl_task` to sleep after adding reactions to a message
 CRAWL_SLEEP = 2
 
+DISCORD_MESSAGE_LINK_RE = re.compile(
+    r"(https?:\/\/(?:(ptb|canary|www)\.)?discord(?:app)?\.com\/channels\/"
+    r"[0-9]{15,21}"
+    r"\/[0-9]{15,21}\/[0-9]{15,21})"
```

Comment:
```suggestion
    r"[0-9]{15,20}"
    r"\/[0-9]{15,20}\/[0-9]{15,20})"
```
```python
>>> int('1'*64, 2)
18446744073709551615
>>> len(str(18446744073709551615))
20
```
The biggest size a Discord snowflake can be is 20.